x/ecocredit/client/tx.go

```diff
@@ -114,7 +114,7 @@ Parameters:
               eg: '[{recipient: "regensdgkjhs2345u79ghisodg", tradable_amount: "10", retired_amount: "2", retirement_location: "YY-ZZ 12345"}]'
               Note: "tradable_amount" and "retired_amount" default to 0.
               Note: "retirement_location" is only required when "retired_amount" is positive.`,
-		Args: cobra.ExactArgs(6),
+		Args: cobra.ExactArgs(0),
```

Comment:
@technicallyty in discord you mentioned maybe updating the usage string to be more clear (removing the square brackets), can you go ahead and update it to be something you think makes more sense?

Do you think:
`gen-batch-json --class-id [class_id] --issuances [issuances] --start-date [start_date] --end-date [end_date] --project-location [project_location] --metadata [metadata]` is more clear?